src/hotspot/share/c1/c1_LIR.cpp

```diff
@@ -838,7 +838,7 @@ void LIR_OpVisitState::visit(LIR_Op* op) {
       assert(opLock->_result->is_illegal(), "unused");
 
       do_stub(opLock->_stub);
-      do_stub(opLock->_throw_imse_stub);
+      do_stub(opLock->_throw_ie_stub);
```

Comment:
Does `_throw_ie_stub` apply to both `lir_lock` & 'lir_unlock`?  I thought only `monitorenter` was being changed.

Reply:
When creating a LIR_OpLock for lir_unlock (in LIR_List::unlock_object(), c1_LIR.cpp:1576), no stub is provided so _throw_ie_stub is null. And do_stub() does nothing if the pointer passed in argument is null.
Did I miss something?

Reply:
You're correct. Thanks for confirming